src/image.rs

```diff
@@ -11,62 +11,68 @@ pub struct TileImage<'c> {
 
 impl<'c> TileImage<'c> {
     pub fn slice_tiles(&self, index: u8) -> Vec<Tile> {
-        let width_in_tiles = self.img.width() / self.config.tilesize;
-        let height_in_tiles = self.img.height() / self.config.tilesize;
-        let morton_idx_max = width_in_tiles * height_in_tiles;
```

Comment:
This is probably fine? There must be a reason why we had the functionality to slice non-square images before (I think maybe we had more ambitious plans for the sub-region stuff) but I don't think it hurts to take it out now.